lib/models/file-info.js

```diff
@@ -33,18 +33,30 @@ function diffHighlight(line) {
 }
 
 FileInfo.prototype.confirmOverwrite = function(path) {
+  var info = this;
```

Comment:
You'r right - no reason/need to save the context there, will update the PR later.